components/apps_form/apps_form_field/apps_form_select_field.tsx

```diff
@@ -222,6 +219,13 @@ export default class AppsFormSelectField extends React.PureComponent<Props, Stat
             selectComponent = this.renderChannelSelect();
             break;
         }
+        return selectComponent;
```

Comment:
This method can be made shorter by returning the select component above, rather than assigning a temp var and using `break` in each spot